multibody/plant/multibody_plant.h

```diff
@@ -1993,6 +1993,24 @@ class MultibodyPlant : public MultibodyTreeSystem<T> {
     return tree().MakeActuatorSelectorMatrix(user_to_joint_index_map);
   }
 
+  /// Returns a vector containing the lower position limits for all joints.
+  /// Any unbounded or unspecified limits are -inf.
+  /// @returns concatenation of each joint's limits, ordered by its
+  /// `JointIndex`.
```

Comment:
Not sure if this is really accurate, since it's really using joint.position_start(), not JointIndex, to determine how joint limits factor into the position limits.